### DISCUSSION TITLE
How to override style of children component when using CSS modules for styling ?

### DISCUSSION THREAD

**Discussion by kieuminhduc01** [vercel/next.js#64722] [Help]
### Summary

I have a nextjs project (version 14.2.1 and using app folder) and I wanna combine combine Tailwind and CSS Stylesheets

Code in **Button** component
```js
import classNames from 'classnames'
import styles from './index.module.css'

const Button = ({ children, className }) => {
  return <button className={classNames(styles.button, className)}>{children}</button>
}
export default Button

```
Code css for **Button** component

```js
.button {
  @apply text-center
  text-red-500;
}
```

Code reference **Button** component above

```js
import Button from '@/components/Common/Button'
import classNames from 'classnames'

export default function Home() {
  return (
    <main className="flex min-h-screen flex-col items-center justify-between p-24">
      <Button className={classNames('text-yellow-300')}>Click me</Button>
    </main>
  )
}
```

I expect button "Click me" has text color is text-yellow-300 but button "Click me" actually has text color is text-red-500

How can I deal with it? Thanhs a lot!

**Comment by icyJoseph** [comment]
Hi,

This is how CSS works! The class that's defined last will win... 

```html
<style>
.blue { color: blue; }
      
.red { color: red; }
</style>

<div class="red blue">Am I red or blue?</div>
```

In this case, it'll be red. In your case, you are banking in the Tailwind CSS ordering. 

For these cases, you need a way to merge them so that in the case of conflicts, the last class name applies:

- https://www.npmjs.com/package/tailwind-merge
